src/js/popup.js

```diff
@@ -717,13 +722,21 @@ Logic.registerPanel(P_CONTAINER_INFO, {
 
         moveTabsEl.parentNode.insertBefore(fragment, moveTabsEl.nextSibling);
       } else {
-        Logic.addEnterHandler(moveTabsEl, async function () {
-          await browser.runtime.sendMessage({
-            method: "moveTabsToWindow",
-            windowId: browser.windows.WINDOW_ID_CURRENT,
-            cookieStoreId: Logic.currentIdentity().cookieStoreId,
-          });
-          window.close();
+        Logic.noOfTabs().then(result => {
```

Comment:
Could you change this to `const numTabs = await Logic.numTabs()` so we don't have to use `then(...)`?

Reply:
> Could you change this to `const numTabs = await Logic.numTabs()` so we don't have to use `then(...)`?

changing it right away!